### PR TITLE
add --color cli option

### PR DESCRIPTION
**Pull request by tavisrudd**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -457,6 +457,16 @@ export function buildArgs(commands = new LazyCommands(), wrapMainHandler = wrapC
             + 'Use --assume-role-arn=no-role to override values in stack-args.yaml and use AWS_* env vars.')
         })
 
+        .option('color', {
+          type: 'string', default: 'auto',
+          choices: ['auto', 'always', 'never'],
+          description: description('whether to color output using ANSI escape codes')
+        })
+        .middleware((args) => {
+          if (args.color === 'never' || (args.color === 'auto' && ! process.stdout!.columns)) {
```
Comment (rymndhng) [inline src/main.ts]: Correct me if I'm wrong, but I don't believe the `!` after `stdout` is valid js

```suggestion
          if (args.color === 'never' || (args.color === 'auto' && ! process.stdout.columns)) {
```

Comment (tavisrudd) [reply src/main.ts]: That’s a typescript feature that is the equivalent of `(process.stdout as any).columns`.

**Comment by tavisrudd** [comment]
I read it as the ‘trust me’ operator.

File: src/main.ts (hunk 1 above)
Comment (tavisrudd) [reply src/main.ts]: I read it as the ‘trust me’ operator.

Comment (tavisrudd) [reply src/main.ts]: I should add that I’d discourage its use on most Unbounce projects that have juniors working on them so there’s a case for using the verbose form even here.

Comment (rymndhng) [reply src/main.ts]: I see! My follow up question: Is the type assertion needed? 

The type of `process.stdout` is `WriteStream`, and `columns` is part of the interface.

Comment (tavisrudd) [reply src/main.ts]: It was last time I worked on tty detection 2 years ago but I didn’t actually check this time. I’ll do that today.

Comment (tavisrudd) [reply src/main.ts]: Hah, no it wasn't required on this. Removed.

**Pull request merged**